doc/installation.rst

```diff
@@ -43,6 +43,7 @@ contribute to the pointpats development.
 
 .. _3.5: https://docs.python.org/3.5/
```

Comment:
Should `3.5` be removed since we are currently only officially supporting [`3.6` & `3.7`](https://github.com/pysal/pointpats/blob/61990426f243f5a8d486be65548d145e5754fb03/setup.py#L61-L62)?

Reply:
Will remove `3.5`. Thanks!